### PR TITLE
Chinese cardboard install guide 

### PR DESCRIPTION
**Pull request by Yang31415926**
This is a Chinese guide for installing cardboard on windows.

### CODE REVIEW

File: guides/pages.yml [hunk 1]
```diff
@@ -60,3 +60,11 @@
   pages:
     - title: "Introduction"
       url: "index"
+
+- title: "Chinese Learning Resources"
+  url: "chinese"
```
Comment (hexixiaozilong) [inline guides/pages.yml]: url: "chinese-learning"

File: guides/chinese/index.md [hunk 2]
```diff
@@ -0,0 +1 @@
+此目录下的文档为中文文档,主要针对以中文为主的使用者解决使用cardstack时遇到的问题
```
Comment (hexixiaozilong) [inline guides/chinese/index.md]: repo: guides/chinese-learning/index.md

File: guides/chinese/cardboard-windows-setup.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
+在使用windows系统进行Cardboard编译时,由于系统的差异性,可能会出现一系统的安装问题,本文档提供一套相对完善的安装示例，并在最大程度上对一些问题进行了规避，使开发者能快速完成平台的运行
```
Comment (hexixiaozilong) [inline guides/chinese/cardboard-windows-setup.md]: Add this at the beginning: 
这个教程是在Windows操作系统上安装Cardstack开源全栈应用[Cardboard](https://github.com/cardstack/cardboard)的流程。同样的安装流程可以应用到其他的开源全站应用上，例如[Cardfolio](https://github.com/cardstack/portfolio)。

File: guides/pages.yml [hunk 4]
```diff
@@ -60,3 +60,11 @@
...
+  pages:
+    - title: "简介"
+      url: "index"
+    - title: "Cardboard的Windows安装步骤"
+      url: "cardboard-windows-setup"
```
Comment (hexixiaozilong) [inline guides/pages.yml]: url: "cardboard-windows-setup-cn"

File: guides/chinese/cardboard-windows-setup.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+ 
+打开%userprofile%/.windows-build-tools,双击执行vs_BuildTools.exe,根据下图勾选的组件进行安装
+
+![vs_build_tools2017](/images/chinese/windows/build_tools2017_5.png)
+ 
+4. 在PowerShell中再次运行`yarn install`命令,即可完成安装
+### MSB4019错误
+执行 yarn install,可能出现 X:\\Microsoft.Cpp.Default.props找不到的情况,此步骤为未设置VCTargetsPath环境
+
+ ![vs_build_tools2017](/images/chinese/windows/default_props_not_found.png)
```
Comment (hexixiaozilong) [inline guides/chinese/cardboard-windows-setup.md]: (/images/chinese-learning/cardboard-windows-setup-cn/default_props_not_found.png)

Use format: /images/section-url/page-url/file-name
This helps differentiating different typpes of files by the article it belongs to.

**Comment by jenweber** [comment]
Looks great, thank you for your work!

**Pull request merged**